src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -2,7 +2,23 @@
 
 public class WorkWithFile {
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String COMMA = ",";
+    public static final String SPACE = " ";
+    public static final int DEFAULT_NUMBER = 0;
+    private final BufferReadWrite buffer = new BufferReadWrite();
+    private final BuyAndSupplyCounter buyAndSupplyCounter = new BuyAndSupplyCounter();
+    private final ReportMessageCreate reportMessageCreate = new ReportMessageCreate();
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        int supplyAmount;
+        int buyAmount;
```

Comment:
why do we declare variables here and initialize them later?
can we move declaration to the initialization?